pkg/vpcmodel/abstractVPC.go

```diff
@@ -246,6 +246,20 @@ type RulesInTable struct {
 	RulesOfType RulesType
 }
 
+// RuleOfFilter a single rule in filter given the layer (SGLayer/NACLLayer)
+type RuleOfFilter struct {
+	LayerName  string             `json:"layer"`
+	FilterName string             `json:"table"`
+	RuleIndx   int                `json:"rule_index"`
```

Comment:
typo `RuleIndx` => `RuleIndex`

Reply:
it was on purpose, not a typo. No big deal  - changed to `RuleIndex`.